src/components/FormAddItem.vue

```diff
@@ -1,10 +1,15 @@
 <template>
-<div class="formadditem">
-  <div class="form-group">
-    <input class="form-control" :value="inputText" @input="changeText" placeholder="Insert task-name..." />
+  <div>
+    <div class="formadditem form-inline">
```

Comment:
css class name should be hyphen separated. Ex: form-add-item instead of formadditem